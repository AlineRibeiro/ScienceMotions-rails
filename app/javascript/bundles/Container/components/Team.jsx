import React from "react";
import teamInfo from "./teamInfo.js";
import teamImgs from "./teamImgs";

const Team = ({}) => (
  <section className="container">
    {teamInfo.map(function(item, key) {
      const imgSrc = teamImgs.find(img => {
        if (img.includes(item.image)) return img;
      });
      return (
        <div className="col-md-3 portfolio-item" key={key}>
          <img
            className="img-responsive"
            style={{
              display: "block",
              marginLeft: "auto",
              marginRight: "auto",
              width: "245px",
              height: "245px",
              marginBottom: "35px"
            }}
            src={imgSrc}
            alt={item.image}
          />
          <p
            style={{
              fontSize: "12px"
            }}
          >
            {" "}
            <span>{item.name}</span> {item.description}
          </p>
        </div>
      );
    })}
  </section>
);

export default Team;
