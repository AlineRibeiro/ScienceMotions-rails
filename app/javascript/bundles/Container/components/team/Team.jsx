import React from "react";

import Monika from "./Monika";

const Team = ({}) => (
  <div>
    <h1>Team</h1>
    <Monika />
  </div>
);

export default Team;
