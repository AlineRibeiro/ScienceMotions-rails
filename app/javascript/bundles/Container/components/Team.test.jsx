import React from 'react';
import renderer from 'react-test-renderer';
import Team from './Team';

describe('Team', () => {
  window.scrollTo = jest.fn(() => ({})); 
  it('renders correctly when there are no items', () => {
    const rendered = renderer.create(<Team />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
})
