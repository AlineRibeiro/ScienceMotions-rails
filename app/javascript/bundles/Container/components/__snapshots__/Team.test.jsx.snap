// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Team renders correctly when there are no items 1`] = `
<div
  id="skrollr-body"
>
  <div
    data-0="background-image: linear-gradient(90deg, hsl(172, 100%, 50%), hsl(318, 68%, 68%)"
    data-450="background-image: linear-gradient(450deg, hsl(172, 100%, 50%), hsl(318, 68%, 68%)"
  >
    <h1>
      Team
    </h1>
    <section
      className="container"
      style={
        Object {
          "paddingTop": "30px",
        }
      }
    >
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="jess"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Jessica Leach
          </span>
           
           is a React developer and illustrator from the UK. When she's stressed she eats bread alone in the bathroom.
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="vanessa"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Vanessa Ho
          </span>
           
           is a Node developer and water technology expert that originally hails from Canada. She also loves cereal. (I mean REALLY loves it.).
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="denise"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Denise Kehoe
          </span>
           
           is an award-winning artist, illustrator and teacher. In her free time she enjoys a superb collection of leggings.
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="kyle"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Kyle Dallafior
          </span>
           
           is a business analyst, developer and ex-teacher, and the best thing to come out of Detroit since Eminem.
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="marie"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Marie Treschow
          </span>
           
           is a back-end developer with a degree in business economics. She's also classy as hell.
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="jerome"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Jerome Minney
          </span>
           
           is a serial entrepreneur and Biz Dev with a background in mechanical engineering and insane dance moves.
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="aakanksha"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Aakanksha Jain
          </span>
           
           is a CSE major undergraduate in Jaipur, India. She likes to code, read, write, draw & since the start of 2018 started contributing to open source.
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="utsha"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Utsha Sinha
          </span>
           
           is an IT sophomore at NSIT, New Delhi, India. Front-end developer with an immense love for music. She's always seen with my laptop and my headphones.
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="sakshi"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Sakshi Sharma
          </span>
           
           is web developer with a strong command of Python, JavaScript, CSS and animation. She loves listening to music and is pursuing a B.Tech from IIIT Vadodara, India.
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="ananya"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Ananya Singh
          </span>
           
           is a code panda - sleeping and eating when not kung-fu coding/ designing/ avoiding people under the guise of listening to music.
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="lauren"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Lauren Barnet
          </span>
           
           works as a microbiologist in Montana, USA and has been coding for the past year. She's taught science in the past and wants to continue to make science accessible.
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="anne-e"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Anne Eckersberg
          </span>
           
           is a visual artist based in Copenhagen, who works with sculpture and digital media in her artistic practice.
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="neer"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Neer
          </span>
           
           is an active rails girls Copenhagen community member. Besides learning to code she likes powerlifting and hiking.
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="monika"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Monika Singh
          </span>
           
           is a full stack developer. She loves loves coding, badminton, exploring new places and always tries to stay uptodate about her interests.
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="ananya"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Ananya Appan
          </span>
           
           is currently studying for an M-Tech at IIIT. She likes to code in Python, C, C++, HTML, CSS, Javascript, Ruby on Rails and React.
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="mili"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Mili Goyal
          </span>
           
           is studying for an M-Tech at IIIT in Bangalore. She's interested in digital design, algorithms and economics.
        </p>
      </div>
      <div
        className="col-md-4 portfolio-item"
      >
        <img
          alt="katrina"
          className="img-responsive"
          style={
            Object {
              "display": "block",
              "height": "245px",
              "marginBottom": "35px",
              "marginLeft": "auto",
              "marginRight": "auto",
              "width": "245px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "12px",
            }
          }
        >
           
          <span>
            Katrina Kennett
          </span>
           
           helps teachers use technology and learners understand the forces of the natural world. She's learning to code to understand what's 'under the hood' of the tools we use everyday.
        </p>
      </div>
    </section>
  </div>
</div>
`;
